### PR TITLE
Adds CSL and General AR model templates

### PR DESCRIPTION
**Pull request by evgeny-izutov**

### CODE REVIEW

**Comment by evgeny-izutov** [comment]
run tests

**Comment by evgeny-izutov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by evgeny-izutov** [comment]
run tests

File: models/action_recognition/model_templates/custom-action-recognition/README.md [hunk 1]
```diff
@@ -75,18 +80,21 @@ where `${DATA_DIR}/train.txt` and `${DATA_DIR}/val.txt` - annotation files where
 <rel_path_to_video_file> <label_id>
 ```
 
+Additionally to enable the CrossNorm augmentation please download the [file](https://storage.openvinotoolkit.org/repositories/openvino_training_extensions/models/custom_action_recognition/mean_std_list.txt) with dumped mean and variance pairs and place it the `${DATA_DIR}`.
```
Comment (Ilya-Krylov) [inline models/action_recognition/model_templates/custom-action-recognition/README.md:83]: place it into

Comment (evgeny-izutov) [reply models/action_recognition/model_templates/custom-action-recognition/README.md]: thanks, fixed

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by evgeny-izutov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by evgeny-izutov** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**